frontend/components/customizer/customizer.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import clipboardCopy from 'clipboard-copy'
-import { staticBadgeUrl } from '../../lib/badge-url'
+import { staticBadgeUrl } from '../../../core/badge-urls/make-badge-url'
```

Comment:
If I'm tracing through things correctly, it seems like this results in eliminating the call to the function in `frontend/lib/resolve-url.js`. I was wondering if this would eliminate the need for `resolve-url` altogether, but see that it's still used in two components (`footer` and `suggestion-and-search`).

Is `resolve-url` something that may eventually change/be consolidated? Or will it continue live on since it's not badge specific?

Reply:
Ugh, I kind of forgot it was still there. May as well take this moment to excavate it. I'd really like to get rid of it.

Reply:
Okay, they're removed.

Suggestions are still broken (see #3033) but I confirmed locally in the debugger that the correct request was being fired and coming back which is what this change affected.

Reply:
Lol, I didn't mean to add more work here (was curious when one should use `resolve-url`) but this certainly makes it simpler!

Reply:
Didn't mean to come across as complaining! This is not a pile of cruft I've been happy with and it's a relief to have it gone. I was hoping I'd found the bottom of it 😁

Part of what makes this hard is that there are a lot of object schemas floating around in the frontend which can make for some tricky debugging. (I also had forgotten about #3033.)

Reply:
> Didn't mean to come across as complaining!

no worries, didn't come across that way at all 👍